spec/controllers/v0/sessions_controller_spec.rb

```diff
@@ -126,10 +134,8 @@
       it 'returns a valid session and user' do
         post :saml_callback
 
-        token = Rack::Utils.parse_query(URI.parse(response.location).query)['token']
-        session = Session.find(token)
-        expect(session).to_not be_nil
-        expect(User.find(session.uuid).attributes).to eq(mvi_user.attributes)
+        expect(created_session).to_not be_nil
+        expect(created_user.attributes).to eq(mvi_user.attributes)
```

Comment:
I couldnt resist fixing up old examples using my new `lets` - but this really isn't important for the PR.